src/blocks/carousel/create-swiper.js

```diff
@@ -2,15 +2,32 @@
  * External dependencies
  */
 import { merge } from 'lodash';
+import { speak } from '@wordpress/a11y';
+import { __, sprintf } from '@wordpress/i18n';
 import Swiper from 'swiper';
 import 'swiper/dist/css/swiper.css';
 
+export const activateSlide = slide => {
+	slide.ariaHidden = 'false';
+	slide.querySelectorAll( 'a' ).forEach( e => ( e.tabIndex = '0' ) );
+};
+
+export const deactivateSlide = slide => {
+	slide.ariaHidden = 'true';
+	slide.querySelectorAll( 'a' ).forEach( e => ( e.tabIndex = '-1' ) );
+};
+
 /**
  * @param {string} container Selector
  * @param {Object} params Params passed to Swiper
  */
-export default function createSwiper( container = '.swiper-container', params = {} ) {
+export const createSwiper = ( container = '.swiper-container', params = {} ) => {
```

Comment:
minor one: It's possible don't use arrow function, just in case. 

```suggestion
function createSwiper( container = '.swiper-container', params = {} ) {
```

Reply:
👍 To make sure I understand, is it just for older browser compatibility?